### DISCUSSION TITLE
Adding mobile app specific redirect URLs to SIWSL

### DISCUSSION THREAD

**Discussion by DriesSchaumont** [simple-login/app#2237] [Q&A] [answered]
Hi!

I was looking at using Simple Login as an identity provider for my Immich instance. Immich provides [documentation](https://immich.app/docs/administration/oauth/#prerequisites) on how to setup OAuth, and suggests to add `app.immich:///oauth-callback` to the Authorized Redirect URLs in order to use OAuth with the mobile app. However, Simple Login blocks these types of URLS being added because they are only allowed to be prefixed with `https://`. 

Am I interpreting this correctly by saying that I would not be able to enable authentication using Simple Loging for immich (at least for use with the mobile app)?
Thanks in advance to anyone who could provide some insight!

**Comment by Fredba** [comment, marked as answer]
Hi, 

You can use this URL : https://example.immich.app/api/oauth/mobile-redirect instead of app.immich:///oauth-callback
In this case you have to activate "MOBILE REDIRECT URI OVERRIDE" in immich "OAuth" configuration panel
